wp-content/themes/twentytwenty-child/archive-student.php

```diff
@@ -38,7 +38,8 @@
             $query->the_post();
             
             $title = str_replace( ' ','-', strtolower( get_the_title() ) );
-            echo '<a href=' . home_url( $title ) . ' class="student-name">' . get_the_title() . '</a>';
+            
+            echo '<div class="student-name"> <a href=' . home_url( $title ) . '>' . get_the_title() . '</a> </div>';
```

Comment:
Shouldn't this lead to the single page of the student.. so `get_permalink()`, instead of `home_url` ?

Reply:
+1 to what @Neychok said ^